src/helpers/get-my-position.js

```diff
@@ -1,10 +1 @@
-import seedableRandom from './seedable-random'
-import {SEED, UNIVERSE_SIZE} from 'constants'
-const {floor} = Math
-
-export default (seed) => [
-  seedableRandom(SEED + 'asdn', 0),
-  seedableRandom(SEED + 'ad', 1)
-]
-  .map((x) => UNIVERSE_SIZE * x)
-  .map(floor)
+export default () => [1000000, 1000000]
```

Comment:
Having gigantic numbers makes the browser wiggle the rendering... That is why I put a much smaller number here.

We can revisit this at a latter point.